source/Octopus.Tentacle/Services/Capabilities/CapabilitiesService.cs

```diff
@@ -9,7 +9,7 @@ public class CapabilitiesService : ICapabilitiesService
     {
         public CapabilitiesResponse GetCapabilities()
         {
-            return new CapabilitiesResponse(new List<string>());
+            return new CapabilitiesResponse(new List<string>{ nameof(IScriptServiceV2)+"Alpha" });
```

Comment:
`IScriptServiceV2` is being added as `Alpha` to allow us to add the service and test it with branch instances of Server while not making it GA until we are happy with the functionality.